todo_app/service/trello_service.py

```diff
@@ -0,0 +1,64 @@
+import requests
+import os
+
+from todo_app.data.item import to_card, CardList
+
+trello = "https://api.trello.com/1/"
+auth = "?key={0}&token={1}".format(os.getenv("TRELLO_API_KEY"), os.getenv("TRELLO_TOKEN"))
+
+to_do_list = CardList([], "ASCENDING")
+
+
+def get_cards_in_list(list_id, status):
+    url = trello + "lists/" + os.getenv(list_id) + "/cards" + auth
+    response = requests.get(url)
+    raw_cards = response.json()
+    parsed_cards = []
+    for card in raw_cards:
+        card["status"] = status
+        parsed_cards.append(to_card(card))
+    return parsed_cards
+
+
+def create_to_do(description):
+    url = trello + "cards/" + auth + "&idList=" + os.getenv("TRELLO_TO_DO") + "&name=" + description
```

Comment:
Format strings are your friend here:
```suggestion
    url = f"{trello}cards/{auth}&idList={os.getenv('TRELLO_TO_DO')}&name={description}"
```